community/curlpp/depends

```diff
@@ -1,2 +1,2 @@
-curl
 cmake
```

Comment:
This should also be `cmake make`

Reply:
thanks cant believe i didnt spot that mistake